crashreports/crashreporter/CrashReportApp.cpp

```diff
@@ -298,8 +303,41 @@ namespace
             mainLayout->Add(commentCtrl, wxSizerFlags().Border(wxALL).Expand());
         }
 
-        if (onSend != nullptr && commentCtrl != nullptr)
+        if (onSend != nullptr)
         {
+            const wxString translatedText = _("See %s for more info.");
+            const wxString translatedLink = _("our Privacy Policy");
```

Comment:
Strings exactly the same as in src/ ?  Check.  Add comment in src/ApplicationPrefs.cpp that it must remain so.

Reply:
i18n-hint comments should work in crashreporter code too (correct extraction to audacity.pot) so you might replicate them